app/client/ui/AdminLeftPanel.ts

```diff
@@ -77,7 +76,7 @@ export function buildAdminLeftPanel(owner: MultiHolder, appModel: AppModel): Pag
       buildPageEntry('docs', 'Page', adminControlsAvailable),
       (adminControlsAvailable ? null :
         cssPanelLink(cssLearnMoreLink(
-          {href: commonUrls.helpAdminControls, target: "_blank"},
+          {href: window.gristConfig.commonUrls.helpAdminControls, target: "_blank"},
```

Comment:
What do you think of defining `app/client/lib/commonUrls.ts`, to export `commonUrl` analogously to `app/server/lib/commonUrls`? Then this code wouldn't change, and more importantly, we'd encapsulate access to the global `window` object.